### PR TITLE
Change devtools to remotes and update repos to facebookincubator

### PR DESCRIPTION
**Pull request by laresbernardo**
- Suggested `remotes` instead of `devtools` because it's lighter and only needed to run `install_github`
- Updated facebookincubator everywhere

### CODE REVIEW

File: doc/GeoLift_Walkthrough.Rmd [hunk 1]
```diff
@@ -28,7 +28,7 @@ You can install `GeoLift` from GitHub using the `devtools` package.
 ## Install devtools if noy already installed
 install.packages("devtools", repos='http://cran.us.r-project.org')
 ## Install GeoLift from github
-devtools::install_github("ArturoEsquerra/GeoLift")
+devtools::install_github("facebookincubator/GeoLift")
```
Comment (NicolasMatrices-v2) [inline doc/GeoLift_Walkthrough.Rmd:31]: Hey @laresbernardo shouldn't this be `remotes::install_github("facebookincubator/GeoLift")` as well?

Comment (laresbernardo) [reply doc/GeoLift_Walkthrough.Rmd]: That's right. Changed in that file as well. Thanks @NicolasMatrices-v2

File: doc/GeoLift_Walkthrough.html [hunk 2]
```diff
@@ -340,11 +340,11 @@ <h1 class="title toc-ignore">GeoLift_Walkthrough</h1>
...
+<p>You can install <code>GeoLift</code> from GitHub using the <code>remotes</code> package.</p>
+<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1"></a><span class="co">## Install remotes if noy already installed</span></span>
+<span id="cb1-2"><a href="#cb1-2"></a><span class="kw">install.packages</span>(<span class="st">&quot;remotes&quot;</span>, <span class="dt">repos=</span><span class="st">&#39;http://cran.us.r-project.org&#39;</span>)</span>
 <span id="cb1-3"><a href="#cb1-3"></a><span class="co">## Install GeoLift from github</span></span>
-<span id="cb1-4"><a href="#cb1-4"></a>devtools<span class="op">::</span><span class="kw">install_github</span>(<span class="st">&quot;ArturoEsquerra/GeoLift&quot;</span>)</span></code></pre></div>
+<span id="cb1-4"><a href="#cb1-4"></a>remotes<span class="op">::</span><span class="kw">install_github</span>(<span class="st">&quot;ArturoEsquerra/GeoLift&quot;</span>)</span></code></pre></div>
```
Comment (NicolasMatrices-v2) [inline doc/GeoLift_Walkthrough.html:347]: @laresbernardo this should probably also change to `facebookincubator/GeoLift` right? Check aat the end of the line.

**Review by NicolasMatrices-v2 — APPROVED** [review]
LGTM! Please  update the DESCRIPTION & CHANGELOG file before merging :)

**Pull request merged**